README.md

```diff
@@ -25,14 +25,16 @@ To start a new worker pool, you can either use `wpool:start_pool` (if you want t
 * **overrun_warning**: The number of milliseconds after which a task is considered *overrun* (i.e. delayed) so a warning is emitted using **overrun_handler**. The default value for this setting is `infinity` (i.e. no warnings are emitted)
 * **overrun_handler**: The module and function to call when a task is *overrun*. The default value for this setting is `{error_logger, warning_report}`.
 * **workers**: The number of workers in the pool. The default value for this setting is `100`
-* **worker**: The [`gen_server`](http://erldocs.com/current/stdlib/gen_server.html) module that each worker will run and the `InitArgs` to use on the corresponding `start_link` call used to initiate it. The default value for this setting is `{wpool_worker, undefined}`. That means that if you don't provide a worker implementation, the pool will be generated with this default one. [`wpool_worker`](http://inaka.github.io/worker_pool/worker_pool/wpool_worker.html) is a module that implements a very simple rfc-like interface
+* **worker_type**: The type of the worker. The available values are `gen_server` and `gen_fsm`. The default value is `gen_server`
```

Comment:
Heads up, http://inaka.github.io/worker_pool/worker_pool/wpool_fsm_worker.html this link is still not working. The documentation needs to be regenerated and updated.

Also,  RFC-like or RPC-like?